client/public/editPodcastPage/editPodcastPage.js

```diff
@@ -0,0 +1,27 @@
+window.onload = (event) => {
+  const deleteButton = document.getElementById('delete-btn')
+  deleteButton.parentNode.removeChild(deleteButton)
+}
+const editPodcast = async () => {
```

Comment:
The name here should be savePodcast I believe

Reply:
Changed.